modules/core/include/opencv2/core/utility.hpp

```diff
@@ -380,7 +380,29 @@ class CV_EXPORTS_W TickMeter
     sumTime = 0;
     counter = 0;
     }
-
+    /**
+    returns frames per second.
+    @param reset_point is value of counter after which TickMeter should reset.
+    */
+    CV_WRAP double getFps(int reset_point = 10)
```

Comment:
can you move implementations to some .cpp file? We would like to keep .hpp files as clean as possible